docs/try-snowplow/accessing-and-querying-your-try-snowplow-data/index.md

```diff
@@ -6,12 +6,72 @@ sidebar_position: 30
 
 ## Accessing your data
 
-Try Snowplow creates a Postgres database where any events you capture are stored. This database contains the standard Snowplow schemas: atomic (for raw data), bad_rows (for data that has failed pipeline validation) and derived (for modeled tables).
+Try Snowplow creates a Postgres database where any events you capture are stored. This database contains the standard Snowplow schemas:
+
+* atomic for raw data
+* bad_rows for data that has failed pipeline validation
+* derived for modeled tables.
+
+Connection details for accessing this data are available in the Try Snowplow console. You will also need to request the password from within the UI.
+
+![](images/1-connectiondetails.png)
 
-Details for accessing this data are available in the Try Snowplow console. You will also need to request the password from within the UI - please bear in mind you will only be able to do this once for security reasons.
 
 ## Querying your data
 
 Like Snowplow BDP, Try Snowplow encourages you to connect your your BI or query tool of choice to access the database and query your data.
 
 You can either copy a sample query from the console tutorial, [check out the Recipes](/docs/recipes/index.md) or start exploring your data with your own queries.
+
+### Example: Querying your data in Supersest / Preset
+
+You can access the PostgreSQL trial database using your preferred BI or SQL query tool.
+If you don’t have permissions to create a new connection in your BI tool, you can use [Preset](https://preset.io/) which is built on the popular open source project, Superset and has a 14 day free trial.
+
+The steps to connect, access and visualize data are below.
+
+1. Select [“Try for Free”](https://preset.io/) on the Preset homepage and enter your details to create an account
+2. Select ‘Yes’ to connect to your team’s database
+
+![](images/2-presetsetup.png)
+
+3. Select PostgreSQL as your database
+
+![](images/3-selectpostgres.png)
+
+4. Enter the connection details provided in the [Try Snowplow UI](https://try.snowplowanalytics.com/access-data): Hostname, Port, Database, Username and Password (request credentials on Try UI if you haven’t done so)
+
+<img src=images/4-connectpostgres.png width="400"/>\
```

Comment:
Resize the image itself and use a markdown image syntax (as discussed on slack)

Reply:
Thanks Nick, updated